RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -215,7 +222,44 @@ MeasurementTrackerEventProducer::updateStrips( const edm::Event& event, StMeasur
   }
 }
 
+//FIXME: just a temporary solution for phase2!
 void 
+MeasurementTrackerEventProducer::updatePhase2OT( const edm::Event& event, Phase2OTMeasurementDetSet & thePh2OTDets ) const {
+
+
+  // Phase2OT Clusters
+  if (pset_.existsAs<std::string>("Phase2TrackerCluster1DProducer")) {
+
+    std::string phase2ClusterProducer = pset_.getParameter<std::string>("Phase2TrackerCluster1DProducer");
+    if( phase2ClusterProducer.empty() ) { //clusters have not been produced
+      thePh2OTDets.setActiveThisEvent(false);
+    } else {
+  
+      edm::Handle<edmNew::DetSetVector<Phase2TrackerCluster1D> > & phase2OTClusters = thePh2OTDets.handle();
+      event.getByToken(thePh2OTClusterLabel, phase2OTClusters);
+      const edmNew::DetSetVector<Phase2TrackerCluster1D>* phase2OTCollection = phase2OTClusters.product();
+  
+      int i = 0, endDet = thePh2OTDets.size();
+      for (edmNew::DetSetVector<Phase2TrackerCluster1D>::const_iterator it = phase2OTCollection->begin(), ed = phase2OTCollection->end(); it != ed; ++it) {
+  
+        edmNew::DetSet<Phase2TrackerCluster1D> set(*it);
+        unsigned int id = set.id();
+        while ( id != thePh2OTDets.id(i)) {
+            ++i;
+            if (endDet==i) throw "we have a problem!!!!";
```

Comment:
actually, I see that it's a copy-paste from other methods in the same class.
OK, no action needed now.